src/Config/PHPIconsConfig.php

```diff
@@ -69,8 +69,9 @@ public function setApiHosts(array $apiHosts): void
     /**
      * @param array<string,string> $localIconSets
      */
-    public function setLocalIconSets(array $localIconSets): void
-    {
+    public function setLocalIconSets(
```

Comment:
I've added a ["quality" github action](https://github.com/yassinedoghri/php-icons/blob/develop/.github/workflows/quality.yml) to check code formatting and linting: it runs ecs, phpstan, and rector.

I think the formatting changes that were applied to the source code here won't comply with the rules.

Reply:
Yes it's weird, I'm gonna check that thanks !